element_lab/export/nwb.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+from datetime import datetime
+from dateutil.tz import tzlocal
+from pynwb import NWBFile
+
+from element_lab import lab
+
+def lab_to_nwb(lab_key):
+    lab_query = lab.Lab & lab_key
+    lab_query = lab_query.join(lab.Lab.Protocol, left=True)
+    lab_query = lab_query.join(lab.Lab.Project, left=True)
+    lab_info = lab.query.fetch1()
```

Comment:
This part won't work if there are multiple Protocols and Projects.

Reply:
Good point! I now separate lab, project, and protocol to allow for many-to-many relationships.